.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,138 @@
+      details as possible._
+
+      [issue search]: ../search?q=is%3Aissue&type=issues
+
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: >-
+      A clear and concise description of what the bug is.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: To Reproduce
+    description: >-
+      Describe the steps to reproduce this bug.
+    value: |
+      1. Have certain environment
+      2. Run given code snippet in a certain way
+      3. See some behavior described
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Expected behavior
+    description: >-
+      A clear and concise description of what you expected to happen.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Logs/tracebacks
+    description: >-
+      |
+      If applicable, add logs/tracebacks to help explain your problem.
+      Paste the output of the steps above, including the commands
+      themselves and their output/traceback etc.
+    render: Python traceback
+  validations:
+    required: true
+
+- type: input
+  attributes:
+    label: Python Version
+    description: Attach your version of Python.
+    render: console
+    value: |
+      $ python --version
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Aiohttp Version
+    description: Attach your version of aiohttp.
+    render: console
+    value: |
+      $ python -m pip show aiohttp
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Multidict Version
+    description: Attach your version of multidict.
+    render: console
+    value: |
+      $ python -m pip show multidict
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Yarl Version
+    description: Attach your version of yarl.
+    render: console
+    value: |
+      $ python -m pip show yarl
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: OS
+    placeholder: >-
+      For example, Arch Linux, Windows, macOS, etc.
+  validations:
+    required: true
+
+- type: dropdown
+  attributes:
+    label: Related component
+    description: >-
+      Aiohttp is both server framework and client library.
+      For getting rid of confusing make sure to select
+      'server', 'client' or 'both'.
+    options:
+      - Server
+      - Client
+      - Both
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Additional context
+    description: |
+      Add any other context about the problem here.
+
+      Describe the environment you have that lead to your issue.
+      This includes proxy server and other bits that are related to your case.
+
+- type: checkboxes
+  attributes:
+    label: Code of Conduct
+    description: |
+      Read the [aio-libs Code of Conduct][CoC] first.
+      [CoC]: https://github.com/aio-libs/.github/blob/master/CODE_OF_CONDUCT.md
```

This will probably not work without an extra line before the link declaration:
```suggestion

      [CoC]: https://github.com/aio-libs/.github/blob/master/CODE_OF_CONDUCT.md
```

---

I am not able to view the rendered view in my fork, to be honest, I don't know why, so I wasn't able to test if there should be an extra line, but I will make the change anyway.

---

> I don't know why

Because it is an alpha feature and GH should flag a repository for it to work. I asked them to add aiohttp, multidict and yarl which is why it works here.